packages/devtools/lib/src/connected_app.dart

```diff
@@ -63,9 +69,14 @@ class ConnectedApp {
     assert(serviceManager.serviceAvailable.isCompleted);
     await serviceManager.isolateManager.selectedIsolateAvailable.future;
 
-    return serviceManager.isolateManager.selectedIsolateLibraries
+    final uris = serviceManager.isolateManager.selectedIsolateLibraries
         .map((ref) => ref.uri)
```

Comment:
nit: You can just replace the `toList().contains(uri)` with `.any((u) => u.startsWith(uri))` which will be faster as well (will only iterate until it finds a match and not create a list)

Reply:
done.